gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/GigawordIndexer.java

```diff
@@ -190,6 +199,11 @@ public static void main(String[] args) {
     builder.properties("running.from.commandline=true");
     builder.bannerMode(Banner.Mode.OFF);
     builder.initializers(new GigawordApplicationContextInitializer());
+
+    // need this line to set user home directory to /.inception
+    // otherwise it will be defaulted to ./webanno and documents will be uploaded here
+    // and Inception app won't be able to find uploaded documents.
+    SettingsUtil.customizeApplication("inception.home", ".inception");
```

Comment:
Where documents are uploaded to will depend on this. Otherwise it defaults to .webanno, and when we run Inception, it will not find the uploaded documents.

Reply:
How does the value of this property translate to a concrete file system path?

Reply:
It updates the string property in the SettingsUtil, which is used by Webanno when setting up repository directory.

Reply:
How does Webanno interpret this value? For example, does it create a sub-directory with that name somewhere? Under the user's home directory or elsewhere?  Can you supply an absolute path? If so, what happens?

Reply:
A file directory is constructed using those properties, like below.
```
new File(System.getProperty(SettingsUtil.getPropApplicationHome(),
                System.getProperty("user.home") + "/"
                        + SettingsUtil.getApplicationUserHomeSubdir()),
                "repository");
```
You can see it here.
https://github.com/webanno/webanno/blob/master/webanno-api-dao/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/dao/RepositoryProperties.java

Reply:
Ok, so this will use the path pointed to by `inception.home` if present and will otherwise fall back on `~/.inception`. Got it